create_box/main.py

```diff
@@ -2,8 +2,29 @@
 
 
 def create_box(height, width, character):
-    pass
+    y = 0
+    box = ''
+    if (width <1) or (height<1):
+        print ('Error, height and width must be >= 1')
+    while y < height:
+        box = (character * width)
+        y +=1 
+    return (box + '\n')* height
 
```

Comment:
If you have the time, to challenge yourself: see if you can figure out how to get create_box() working without using a loop. It can be done but it might be a little tricky. I'm pretty sure you'll be able to figure it out though :P

Another thing about your code. It works, but there are a few things you can change...
Every time your while loop runs the code inside it (for as long as y < height) you are resetting box. You say `box = (character * width)`, so every time your loop goes through your essentially just doing the same thing and nothing is changing.

I see what you're trying to do here though. Inside of your while loop you could have had something like this: 
```
while y < height:
         box += (character * width) + '\n'
         y +=1 
return box
```
In the code above, each time the while loop goes through it add's the line (which is the character*width) then it increases y and the loop runs as long as y is less than the height. Once y reaches the height, your loop will stop and `box` will have as many lines(character*width) as the height-- so then you'd have your completed box and we just return it after that.

Does all of that make sense? PM me if you have any questions